message_chunker.py

```diff
@@ -0,0 +1,33 @@
+import nltk
+
+def message_chunk(command):
+    words = nltk.word_tokenize(command)
+    taggedTokens = nltk.pos_tag(words)
```

Comment:
camelCase is preferred in JS, while underscores are preferred in Python. Please use underscores for names instead of camelCase in this file as well as in server.py!